src/meta/proto-conv/src/data_from_to_protobuf_impl.rs

```diff
@@ -149,6 +149,7 @@ impl FromToProto for dv::DataTypeImpl {
                     ver: VER,
                     min_reader_ver: MIN_READER_VER,
                     dt: Some(Dt::NullType(pb::Empty {})),
+                    dt24: None,
```

Comment:
we should change the file name to help people know it's all about `Dt24Version` table meta.

Reply:
Maybe we could freeze and remove the datavalues crate  then using the latest commit in cargo file starting from this pr.

Reply:
> we should change the file name to help people know it's all about `Dt24Version` table meta.

What file name? Only part of this file changed.